README.md

```diff
@@ -2,13 +2,12 @@
 
 [![License](https://img.shields.io/github/license/Qiskit/qiskit-experiments.svg?style=popout-square)](https://opensource.org/licenses/Apache-2.0)
 
-**This repo is still under active development, there will be breaking API changes**
+**This repo is still under active development, there will be breaking API changes.**
 
-**Qiskit** is an open-source framework for working with noisy quantum computers
-at the level of pulses, circuits, and algorithms.
-
-Qiskit Experiments is a repository that builds tools for building, running,
-and analysis of experiments on noisy quantum computers.
+**Qiskit Experiments** is a repository that builds tools for building, running,
+and analyzing experiments on noisy quantum computers using Qiskit. You can see the 
+most up-to-date documentation corresponding to the main branch of this repository 
+[here](https://qiskit.org/documentation/experiments/dev/).
```

Comment:
I don't know if it is helpful or confusing to point to the stable docs as well? There are kind of two audiences, people just using the current release and advanced users trying to develop new features or check out a feature before release.

Reply:
I checked a few Qiskit repositories but there was no standard. My thinking was that people who are on the github page are probably more interested in browsing the dev docs rather than the stable docs, and there's also a link from the dev docs to the stable ones, but having both links on github is probably better than not.